### DISCUSSION TITLE
Potentially mismatch between current ontology version and docs?

### DISCUSSION THREAD

**Discussion by GeorgFerdinandSchneider** [qudt/qudt-public-repo#125] [Q&A] [answered]
Dear QUDT-Team,

thanks for providing this excellent ontology.

Their potentially might be some outdated documentation?

cf.

> http://qudt.org/doc/2019/10/DOC_VOCAB-UNITS-ALL-v2.1.html

The instance

>http://www.qudt.org/2.1/vocab/unit/KiloN

is missing.

Best

Georg

**Comment by steveraysteveray** [comment, marked as answer]
Thanks for your comment.
Actually, that is by design. We don't provide resolution of *versioned* instance URIs, just *unversioned* ones. So http://www.qudt.org/vocab/unit/KiloN *does* resolve (both as an html and a ttl file, depending on your request).
We do provide resolution of both versioned and unversioned graphs (i.e. http://www.qudt.org/2.1/vocab/unit and http://www.qudt.org/vocab/unit). This is so that applications can import a specific version if desired.

And as a final note, we are now up to http://www.qudt.org/doc/2020/05/DOC_VOCAB-UNITS-ALL-v2.1.html on our web site, which contains many more units than the 2019/05 version you referenced. Please see qudt.org, which will lead you to the latest web page. (The absolute latest version of the model is always on this github repository, which periodically gets published on the website).

**Comment by GeorgFerdinandSchneider** [comment]
@steveraysteveray thank you for this clarification!
